cmd/ipfs/daemon.go

```diff
@@ -510,6 +512,23 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 		fmt.Println("(Hit ctrl-c again to force-shutdown the daemon.)")
 	}()
 
+	// Give the user heads up if daemon running in online mode has no peers after 1 minute
+	if !offline {
+		time.AfterFunc(1*time.Minute, func() {
+			ipfs, err := coreapi.NewCoreAPI(node)
+			if err != nil {
+				log.Errorf("failed to access CoreAPI: %v", err)
+			}
+			peers, err := ipfs.Swarm().Peers(cctx.Context())
+			if err != nil {
+				log.Errorf("failed to read swarm peers: %v", err)
+			}
+			if len(peers) == 0 {
+				log.Error("failed to bootstrap (no peers found): consider updating Bootstrap or Peering section of your config")
```

Comment:
```suggestion
				log.Warn("no peers found during bootstrap: consider updating the Bootstrap or Peering section of your config")
```
This is more appropriate to log as a warning, as there is not an actual failure but probably indicates something is wrong.

Reply:
Afraid we need to keep Error to guarantee this to be printed to the stdout with default log settings, but I agree we should  warn instead of error when both sections are empty – see https://github.com/ipfs/go-ipfs/pull/8166/commits/904cc737162eed91ff5a172ac65c758387da9100